here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/AbstractTask.java

```diff
@@ -488,13 +490,8 @@ protected final void unlock() {
         state.set(State.DONE);
         final long newValue = AbstractTask.threadCount.decrementAndGet();
         assert newValue >= 0L;
-        String author = context.getAuthor();
-        if (author != null) {
-          AtomicInteger authorUsage = AbstractTask.authorUsageMap.getOrDefault(author, new AtomicInteger(0));
-          if (authorUsage.get() > 0) {
-            decrementAuthorUsage(author);
-          }
-        }
+        String principal = getPrincipalName();
```

Comment:
We can simply use getPrincipalName() ? (variable principal can be removed)

Reply:
Not applicable.